packages/core/src/graph/graph.ts

```diff
@@ -1426,6 +1479,8 @@ export default abstract class AbstractGraph extends EventEmitter implements IAbs
       });
     }
 
+    this.positionsAnimate(animateCfg);
```

Comment:
I guess we have to check if animation flag is on here ? Else resort to [refreshPositions()](https://g6.antv.vision/en/docs/api/graphFunc/item#graphrefreshpositions)

Reply:
the animate flag is always external to the GraphAnimateConfig, but I agree that for consistency with the other functions it should be possible to disable the animation